py-polars/polars/lazyframe/frame.py

```diff
@@ -225,8 +225,9 @@ def _gpu_engine_callback(
         install_message=(
             "Please install using the command "
             "`pip install cudf-polars-cu12` "
-            "(or `pip install --extra-index-url=https://pypi.nvidia.com cudf-polars-cu11` "
-            "if your system has a CUDA 11 driver)."
+            "(CUDA 12 is required for RAPIDS cuDF v25.08 and later). "
+            "If your system has a CUDA 11 driver, install with "
+            "`pip install --extra-index-url=https://pypi.nvidia.com cudf-polars-cu11==25.06` "
```

Comment:
It's not technically in scope for this PR, but we may as well take this opportunity to update the Polars docs with the fact that we don't need our index anymore.
```suggestion
            "`pip install cudf-polars-cu11==25.06` "
```